UnitTests/Version.cpp

```diff
@@ -66,3 +66,52 @@ TEST(Version, NotEqualsOperator)
     EXPECT_TRUE((Version{2, 1, 3} != Version{2, 3, 1}));
     EXPECT_TRUE((Version{2, 1, 3} != Version{3, 1, 2}));
 }
+
+TEST(Version, LessThanOperator)
+{
+    EXPECT_FALSE((Version{} < Version{}));
+    EXPECT_TRUE((Version{0, 0, 0} < Version{0, 1, 0}));
+    EXPECT_TRUE((Version{2, 4, 1} < Version{4, 0, 10}));
+}
+
+TEST(Version, LessThanEqualToOperator)
+{
+    EXPECT_TRUE((Version{} <= Version{}));
+    EXPECT_TRUE((Version{0, 0, 0} <= Version{0, 1, 0}));
+    EXPECT_TRUE((Version{2, 4, 1} <= Version{4, 0, 10}));
+}
+
+TEST(Version, GreaterThanOperator)
+{
+    EXPECT_FALSE((Version{} > Version{}));
+    EXPECT_FALSE((Version{0, 0, 0} > Version{0, 1, 0}));
+    EXPECT_FALSE((Version{2, 4, 1} > Version{4, 0, 10}));
+}
+
+TEST(Version, GreaterThanEqualToOperator)
+{
+    EXPECT_TRUE((Version{} >= Version{}));
+    EXPECT_FALSE((Version{0, 0, 0} >= Version{0, 1, 0}));
+    EXPECT_FALSE((Version{2, 4, 1} >= Version{4, 0, 10}));
```

Comment:
Generally I dislike "magic-numbers" but not enough in test code to avoid these in it. I'm gonna update this review action to only review library code so stuff like this is skipped in the future.